src/js/providers/tracks-mixin.js

```diff
@@ -98,7 +98,7 @@ define(['../utils/underscore',
 
         if (this._textTracks) {
             if (this._renderNatively) {
-                this.addTracksListener(this._textTracks, 'change', textTrackChangeHandler.bind(this));
+                this.addTracksListener(this.video.textTracks, 'change', textTrackChangeHandler.bind(this));
```

Comment:
I agree that we should eventually find a way to consolidate the video tag tracks and _textTracks to be one and the same, or group them in a way to get what we need when we need it.